wizard/src/utils/detect_place.ts

```diff
@@ -25,7 +25,7 @@ import {
 import countriesJSON from "./country_mappings.json";
 import statesJSON from "./state_mappings.json";
 
-const MIN_HIGH_CONF_DETECT = 0.7;
+const MIN_HIGH_CONF_DETECT = 0.4;
```

Comment:
yeah, it's because of those non-country abbreviations. we can discuss with the team but for now Guha said he'd be fine with a super low threshold to make sure we are unblocked on Monday.